packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -492,40 +492,24 @@ mixin WidgetsBinding
       _debugShowWidgetInspectorOverrideNotifierObject ??= ValueNotifier<bool>(false);
   ValueNotifier<bool>? _debugShowWidgetInspectorOverrideNotifierObject;
 
-  /// The default behavior for taps in the app when the widget inspector is
-  /// enabled.
+  /// The notifier for whether or not taps on the device are treated as widget
+  /// selections when the widget inspector is enabled.
   ///
   /// - If true, taps in the app are intercepted by the widget inspector.
   /// - If false, taps in the app are not intercepted by the widget inspector.
-  bool get debugWidgetInspectorDefaultTapBehaviorEnabled {
-    return debugWidgetInspectorDefaultTapBehaviorEnabledNotifier.value;
-  }
-
-  /// Sets the default behavior for taps in the app when the widget inspector is
-  /// enabled.
-  ///
-  /// See [debugWidgetInspectorDefaultTapBehaviorEnabled].
-  set debugWidgetInspectorDefaultTapBehaviorEnabled(bool value) {
-    debugWidgetInspectorDefaultTapBehaviorEnabledNotifier.value = value;
-  }
-
-  /// The notifier for the default behavior for taps in the app when the widget
-  /// inspector is enabled.
-  ///
-  /// See [debugWidgetInspectorDefaultTapBehaviorEnabled].
-  ValueNotifier<bool> get debugWidgetInspectorDefaultTapBehaviorEnabledNotifier =>
-      _debugWidgetInspectorDefaultTapBehaviorEnabledNotifierObject ??= ValueNotifier<bool>(true);
-  ValueNotifier<bool>? _debugWidgetInspectorDefaultTapBehaviorEnabledNotifierObject;
+  ValueNotifier<bool> get debugWidgetInspectorSelectionOnTapEnabledNotifier =>
```

Comment:
should the public one be a ValueListenable? If not, can we just use a single public notifier instead of having a public getter that returns the private notifier (which essentially makes the private notifier public)?

Reply:
Yes, switched to ValueListenable!